src/docs/truffle/reference/configuration.md

```diff
@@ -131,6 +131,24 @@ For each network, you can specify `host` / `port`, `url`, or `provider`, but not
 - https://sandbox.truffleteams.com/yoursandboxid
 - wss://sandbox.truffleteams.com/yoursandboxid
 
+#### Special (managed) networks: `test` and `develop`
+
+The networks `test` and `develop` are special, in that they can be used to specify the settings for the managed Ganache instance started by `truffle test` and `truffle develop`, respectively.  These options are simply included like any other network options.  The supported options are:
+
+* `network_id`: The network ID to use.  Defaults to 5777.
+* `total_accounts`: The number of unlocked accounts Ganache should provide.  Defaults to 10.  Alias: `accounts`.
+* `default_balance_ether`: The default amount of ether to seed unlockd accounts with, specified in ether.  Defaults to 100 ether.  Alias: `defaultEtherBalance`.
+* `blockTime`: If specified, the number of seconds inbetween mining blocks (as opposed to automatically mining on each transaction sent).  Defaults to 0 (i.e., automine).
+* `fork`: If specified, the URL of a provider that Ganache should fork from.
+* `hardfork`: Allows specifying the hardfork (EVM version) that Ganache should use.
+* `gasLimit`: The block gas limit.  Defaults to 6721975.
+* `gasPrice`: The gas price, specified in wei.  Defaults to 2000000000 wei.
```

Comment:
A style observation, please feel free to ignore. These numbers are hard to process quickly. Especially with large wei values. 6_721_975 and 2_000_000_000 is easier.